.asf.yaml

```diff
@@ -3,3 +3,29 @@ notifications:
     issues:       notifications@dubbo.apache.org
     pullrequests: notifications@dubbo.apache.org
     jira_options: link label link label
+github:
+    homepage: https://dubbo.apache.org/
+    description: "Go Implementation For Apache Dubbo ."
+    labels:
+        - go
+        - rpc
+        - microservices
+        - http2
+        - service-mesh
+    features:
+        # Enable wiki for documentation
+        wiki: true
+        # Enable issue management
+        issues: true
+        # Enable projects for project management boards
+        projects: true
+    protected_branches:
+        master:
+            # only disable force push
+            foo: bar

```

Comment:
Is it possible to enable some extra rules here? Like merging a PR with at least three approves and CIs passed?

Reply:
Yes, it is possible to add those restrictions, but I personally don't think it's necessary to add that for the current state of our project. Giving more flexibility is more important now.

Also I guess this should not be a blocker for this pull request.